manufacturing/blower/README.md

```diff
@@ -92,3 +109,18 @@ You will have to crimp connectors to the second end of the wiring that connects
 to the [PCB](../../pcb).
 It depends on your build layout, but in the case that you need longer wires, you will have to crimp both ends.
 In such a case you might need a greater number of XH and VH connector components listed in the BOM above.
+
+## Assembly Instructions of Blower to the Enclosure
+
+**STEP:1** Attach the Blower Driver to the Driver mounting plate and fix the assembly to the Blower holding plate and Adapter Mounting plate by using 1" spacers for the blower wires to connect the driver.
```

Comment:
Why not link to parts of this document where those parts are documented. And actually, why not mention parts by BOM number like on the pinch valve page?